### DISCUSSION TITLE
Open4es Shader Android Beta 7 VGPU & PC

### DISCUSSION THREAD

**Discussion by M4hir2008** [Open4Es/Open4Es-Shader-Android#134] [General]
This Is 2nd Release of VGPU and Works Best on Snapdragon Prossecors.
Changes:
Added 3 profiles- Low,Normal,High
Changed World Coloring
Changed Some option Names
Depth of Field Now can be turned on (Fixed shader.properties becauseDepth of field wasn't working in beta-6)

<hr /><em>This discussion was created from the release <a href='https://github.com/Open4Es/Open4Es-Shader-Android/releases/tag/1.1.0-Beta-7-VGPU'>Open4es Shader Android Beta 7 VGPU & PC</a>.</em>

**Comment by Billybobertjoe** [comment]
Will this work on questcraft?

**Comment by ENDERMANYK** [comment]
no idea since we dont have the device to test

**Reply by Billybobertjoe** [reply]
How do i install this

**Comment by Billybobertjoe** [comment]
Can I use this on either 1.19.3 or 1.18.2?

**Comment by Billybobertjoe** [comment]
Can I use this with fabric without mods?

**Reply by M4hir2008** [reply]
bro its not a big project. it doesn't have any modding support. and not sure with fabric and iris but it will work with optifine and forge and just optifine. also u can try optifabric. i would recommend 1.16.5 and bellow since the shader is written in #version 120

**Comment by dj-jinsun** [comment]
1.16.5  and 1.15.2 can？
